.github/workflows/typescript-coverage.yml

```diff
@@ -18,12 +18,14 @@ jobs:
     env:
       NPM_REGISTRY_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - uses: ./.github/actions/prepare-environment
 
       - name: Lerna bootstrap
-        run: npx lerna bootstrap
+        run: npx lerna bootstrap --since=$(git merge-base --fork-point origin/dev)
```

Comment:
А мы же тут можем использовать ${{  github.ref  }}  и нам по идеи не нужно искать fork-point потому что это ровно он

то есть тупо `--since=${{ github.ref }}` и тогда тебе fetch-depth не нужен наверно

Reply:
@Yeti-or Попробовал твой вариант

https://github.com/salute-developers/plasma/actions/runs/5373895416/jobs/9748711481

```bash
lerna notice cli v6.5.1
lerna verb rootPath /home/runner/work/plasma/plasma
lerna info versioning independent
lerna info ci enabled
lerna notice filter changed since "refs/pull/5[7](https://github.com/salute-developers/plasma/actions/runs/5373895416/jobs/9748711481#step:4:8)4/merge"
lerna verb hasTags true
lerna info Looking for changed packages since refs/pull/574/merge
lerna info ignoring diff in paths matching [ '*.md', '*.component-test.tsx', 'package-lock.json' ]
lerna ERR! Error: Command failed with exit code 12[8](https://github.com/salute-developers/plasma/actions/runs/5373895416/jobs/9748711481#step:4:9): git diff --name-only refs/pull/574/merge -- actions/platform-test
lerna ERR! fatal: bad revision 'refs/pull/574/merge'
lerna ERR!     at makeError (/home/runner/work/plasma/plasma/node_modules/execa/lib/error.js:60:11)
lerna ERR!     at Function.module.exports.sync (/home/runner/work/plasma/plasma/node_modules/execa/index.js:1[9](https://github.com/salute-developers/plasma/actions/runs/5373895416/jobs/9748711481#step:4:10)4:17)
lerna ERR!     at Object.execSync (/home/runner/work/plasma/plasma/node_modules/@lerna/child-process/index.js:39:16)
lerna ERR!     at diffSinceIn (/home/runner/work/plasma/plasma/node_modules/lerna/dist/cli.js:388:24)
lerna ERR!     at hasDiffSinceThatIsntIgnored (/home/runner/work/plasma/plasma/node_modules/lerna/dist/cli.js:361:18)
lerna ERR!     at isCandidate (/home/runner/work/plasma/plasma/node_modules/lerna/dist/cli.js:445:77)
lerna ERR!     at /home/runner/work/plasma/plasma/node_modules/lerna/dist/cli.js:282:9
lerna ERR!     at Map.forEach (<anonymous>)
lerna ERR!     at collectPackages (/home/runner/work/plasma/plasma/node_modules/lerna/dist/cli.js:281:12)
lerna ERR!     at collectUpdates (/home/runner/work/plasma/plasma/node_modules/lerna/dist/cli.js:444:[10](https://github.com/salute-developers/plasma/actions/runs/5373895416/jobs/9748711481#step:4:11))
lerna ERR! lerna Command failed with exit code [12](https://github.com/salute-developers/plasma/actions/runs/5373895416/jobs/9748711481#step:4:13)8: git diff --name-only refs/pull/574/merge -- actions/platform-test
lerna ERR! lerna fatal: bad revision 'refs/pull/574/merge'
```

Reply:
@Yeti-or 

1) Попробовал с указанием `${{ github.base_ref }}` в нашем случае **dev** - тоже сломалось.
2) Попробовал с `fetch-depth: 100`, тогда lerna bootstrap начинает все пакеты собирать.

Reply:
печалька